erpnext/accounts/page/pos/pos.js

```diff
@@ -1639,6 +1639,14 @@ erpnext.pos.PointOfSale = erpnext.taxes_and_totals.extend({
 		this.si_docs = this.get_doc_from_localstorage();
 		if (this.name) {
 			this.update_invoice()
+			//to retrieve and set the default payment
+			invoice_data[this.name] = this.frm.doc;
+			invoice_data[this.name].payments[0].amount = this.frm.doc.net_total
+			invoice_data[this.name].payments[0].base_amount = this.frm.doc.net_total
```

Comment:
This is already implemented
https://github.com/frappe/erpnext/blob/develop/erpnext/public/js/controllers/taxes_and_totals.js#L614-L617